app/views/resume_mailer/send_email.text.erb

```diff
@@ -0,0 +1,11 @@
+Resumes!
+
+If you're looking for a developer, please find links to our resumes below.
+
+Backend Devs:
+<%= link_to "Kam's resume", "https://docs.google.com/document/d/1jDm8WwZO2a4wnZOiVdQCy81WkYUhmUIbwFmCeZQVaxo/edit?usp=sharing" %>
+<%= link_to "Paul's resume", "https://docs.google.com/document/d/1UrFqCa0Nou1NegFLIEedpwA8BufKEOha/edit?usp=sharing&ouid=113327344471120951554&rtpof=true&sd=true" %>
+<%= link_to "Ed's resume", "https://docs.google.com/document/d/1hOmXy9700lKb3qPqsViEZ1QQJpxspTWm7yhfhKmK7ug/edit?usp=sharing" %>
+
+Frontend Devs:
+<%= link_to "Arden's resume", "https://docs.google.com/document/d/1R_ovsAfjZhQoLMkUBTFzz66M54nK953uDNt_Z--NjYE/edit?usp=sharing" %>
```

Comment:
I thought it was intersection that you created templates for both html and plain text. Is that simply to cover all the bases, or is it more about the way in which ActionMailer builds the messages?